lib/calendar_assistant/has_duration.rb

```diff
@@ -8,6 +8,11 @@ module ClassMethods
       def duration_in_seconds start_time, end_time
         (end_time.to_datetime - start_time.to_datetime).days.to_i
       end
+
+      def cast_date(date, time_zone = Time.zone.name)
```

Comment:
`cast_date` feels like an incomplete name, since it also handles times (or datetimes). `case_date_or_datetime` is long but says what it does. Any other ideas?

Reply:
let's go with `case_date_or_datetime`

Reply:
See my comment below - this method doesn't actually take a date, so I think I'd vote for `cast_datetime`.